File: Java/Program/Clothing.java
class Clothing extends Produk // anaknya
{
	private String size,material;
	private char gender;
	
	// construct
	public Clothing()
	{
		this.size = "";
		this.material = "";
		this.gender = ' ';
	}
	
	public Clothing(String size, String material, char gender)
	{
		this.size = size;
		this.material = material;
		this.gender = gender;
	}
	
	/* getter*/
	public String getSize()
	{
		return this.size;
	}
	
	public String getMaterial()
	{
		return this.material;
	}
	
	public char getGender()
	{
		return this.gender;
	}
	
	/* setter */
	public void setSize(String size)
	{
		this.size = size;
	}
	
	public void setMaterial(String material)
	{
		this.material = material;
	}
	
	public void setGender(char gender)
	{
		this.gender = gender;
	}
}

File: C++/Program/Clothing.cpp
#include "Produk.cpp"

using namespace std;

class Clothing : public Produk
{
	private: 
		string size,material;
		string gender;
		
	public:
		// construct
		Clothing()
		{
			this->size = "";
			this->material = "";
			this->gender = "-";
		}
		
		Clothing(string size, string material, string gender)
		{
			this->size = size;
			this->material = material;
			this->gender = gender;
		}
		
		/* getter */
		string getSize()
		{
			return this->size;
		}
		
		string getMaterial()
		{
			return this->material;
		}
		
		string getGender()
		{
			return this->gender;
		}
		
		/* setter */
		void setSize(string size)
		{
			this->size = size;
		}
		
		void setMaterial(string material)
		{
			this->material = material;
		}
		
		void setGender(string gender)
		{
			this->gender = gender;
		}
};